Makefile

```diff
@@ -61,22 +55,22 @@ Cargo.lock: Cargo.toml rs/Cargo.toml
 rs: Cargo.lock
 	$(CARGO_TEST)
 
-.PHONY: godep
-godep: Gopkg.toml $(DEP)
-	$(DEP) ensure
+.PHONY: go-build
+go-build:
+	$(GO) build ./...
 
 .PHONY: go
-go: godep $(PROTOC)
-	$(GO) install ./vendor/github.com/golang/protobuf/protoc-gen-go
-	rm -rf go/*
+go: go-build $(PROTOC)
+	$(GO) install github.com/golang/protobuf/protoc-gen-go
 	$(PROTOC_GO) proto/destination.proto
 	$(PROTOC_GO) proto/http_types.proto
 	$(PROTOC_GO) proto/identity.proto
 	$(PROTOC_GO) proto/net.proto
 	$(PROTOC_GO) proto/tap.proto
+	@rm -rf github.com/
```

Comment:
It looks like this deletes all of the generated code?

Reply:
This has been moved to a `move-proto` target as part of 18bc5dc